src/main/java/com/ibm/dfdl/sample/sax/reader/DFDLToSAXEventAdapter1.java

```diff
@@ -31,7 +31,8 @@
 	@Override
 	public void elementValue(String value, DFDLSchemaType baseSchemaType) throws DFDLUserException {
 		StringBuilder sb = new StringBuilder();
-		String escapedValue = XMLUtils.escape(value, sb).toString();
+		String valueWithCRLFsToLFAndPUAs = XMLUtils.remapXMLIllegalCharactersToPUA(value);
```

Comment:
Calling the same thing here that Daffodil does for constructing these XML infosets.